client/file_logging.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+
+import king_phisher
+import king_phisher.client.plugins as plugins
+import king_phisher.client.gui_utilities as gui_utilities
+
+# logger name value
+LOGGER_NAME = ''
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Zach Janice']
+	title = 'Logger'
+	description = """
+	Keep logs of campaign feedback and results. The directory
+	of the logged file(s), the file size (in MB) of the log, and
+	the number of log files kept can be specified. Plugin requires
```

Comment:
No they can't, it's also not necessary to specify that the plugin requires builtin Python packages.